exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -83,6 +83,16 @@
       Submit
     </button>
     <input type="checkbox" name="forceLimit" value="limit" <#if model.isAutoLimitEnabled()>checked</#if>> Limit results to <input type="text" id="autoLimit" name="autoLimit" min="0" value="${model.getDefaultRowsAutoLimited()?c}" size="6" pattern="[0-9]*"> rows <span class="glyphicon glyphicon-info-sign" title="Limits the number of records retrieved in the query. Ignored if query has a limit already" style="cursor:pointer"></span>
+    <label> Store Format
```

Comment:
We should remove this part since store format names depend on the file storage plugin configuration and current default schema thus they cannot be hard-coded.

Reply:
This should work for 99% of users, I think.  If they select something that isn't supported they will just get an error.  The alternative would be a free-form text input, which would still just report an error if you input an unsupported value.  This drop-down can be updated over time to include all known supported output formats of all the plugins and the user can more easily pick a format using a drop-down than a text input.